src/components/app/MenuButtons/Permalink.css

```diff
@@ -2,6 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+.menuButtonsPermalinkButtonButton::before {
```

Comment:
These class names are crazy, I'll do a pass on them in a next PR. Previously we had this `menuButtons` wrapper that we don't have anymore, so I think there's room to simplification.